hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/SCMContainerManager.java

```diff
@@ -293,6 +307,12 @@ public void deleteContainer(ContainerID containerID) throws IOException {
                 " it's missing!", containerID);
       }
       scmContainerManagerMetrics.incNumSuccessfulDeleteContainers();
+    } catch (NodeNotFoundException nnfe) {
+      scmContainerManagerMetrics.incNumFailureDeleteContainers();
+      throw new SCMException(
+          "Failed to delete container " + containerID + ", reason : " +
+              "node doesn't exist.",
```

Comment:
Can we add the node info to the Exception? Is it part of the NodeNotFoundException?